servers/legendnation/metadata.json

```diff
@@ -0,0 +1,47 @@
+{
+    "id": "legendnation",
+    "name": "LegendNation",
+    "store": "https://store.legendnation.net/",
+    "website": "https://legendnation.net/",
+    "minecraftVersions": [
```

Comment:
Add your versions like this:

    "minecraftVersions": [
        "1.7.*",
        "1.8.*",
        "1.12.*",
        "1.16.*",
        "1.17.*",
        "1.18.*",
        "1.19.*",
        "1.20.*"
    ],